### PR TITLE
L2-193: Readme NNS Svelte Project

### PR DESCRIPTION
**Pull request by lmuntanerDfinity**
# Motivation

Describe the Svelte project in the Readme file.

# Changes

* Readme in svelte directory.

# Tests

No tests.

### CODE REVIEW

**Comment by lmuntanerDfinity** [comment]
@peterpeterparker @mstrasinskis @nmattia please check the new readme and let me know what you think.

Once we deploy a page, we should keep the readme up to date.

**Review by peterpeterparker — APPROVED** [review]
Sure works for me. Maybe wait for another review, there are better english speaker than me 😉

Regarding mentioning the status, it implies that we also update the README time to time. Just a thing to keep in mind.

Later on it would maybe be also interesting to add a "contributing section and guidelines" in the top README of the repo, just an idea, not related to this PR.

**Comment by lmuntanerDfinity** [comment]
I also like the idea of the contributing, but that part probably should be at the top readme in the repository.

**Review by nmattia — APPROVED** [review]
LGTM! Just a bit curious about `NNS Dapp`, `NNS-JS`, `Agent-JS`, I think I've always seen those written in lowercase. Should we change that for consistency? Or is the uppercase version the official one?

File: frontend/svelte/README.md [hunk 1]
```diff
@@ -1,107 +1,70 @@
...
-
-_Note that you will need to have [Node.js](https://nodejs.org) installed._
+The plan is to deploy the pages of this project one by one as soon they are ready.
 
-## Get started
+- Log in page: Deployed.
+- Proposals: In testing.
+- Neurons: In development.
+- Accounts: In development.
+- Canisters: In development.
```
[the remark this answers is not in the document]
Comment (lmuntanerDfinity) [reply frontend/svelte/README.md:20]: But I think it's interesting information, isn't it? Maybe we can skip the "In testing".

[the remark this answers is not in the document]
Comment (nmattia) [reply frontend/svelte/README.md:20]: Up to you! Just gotta make sure it's kept up to date otherwise it'll just be confusing.

**Comment by lmuntanerDfinity** [comment]
@nmattia good point, I changed to use lowercase for nns-js and agent-js. For NNS Dapp, I found that we use "NNS Dapp" mainly in public facing docs. So, I am keeping that one.

**Pull request merged**